nautobot_device_lifecycle_mgmt/models.py

```diff
@@ -49,6 +49,7 @@ class HardwareLCM(PrimaryModel):
     end_of_support = models.DateField(null=True, blank=True, verbose_name="End of Support")
     end_of_sw_releases = models.DateField(null=True, blank=True, verbose_name="End of Software Releases")
     end_of_security_patches = models.DateField(null=True, blank=True, verbose_name="End of Security Patches")
+    last_modified_date = models.DateField(null=True, blank=True, verbose_name="Vendor Last Updated")
```

Comment:
I would change `verbose_name` to "Last Updated by Vendor" to make it consistent with other views.